apitester/apitester.script

```diff
@@ -51,27 +51,88 @@ if fbinstant.mock then
 end
 
 
+local function check_init()
```

Comment:
What if we do this check inside the extension in the JS side of fbinstant.initialize()? This way you always call fbinstant.initialize() in your game code, but in the case that Module._instant_inited has been set "from the outside" (ie index.html) it will invoke the callback immediately.

Reply:
I'll let you do this